<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private 'security.authentication.success_handler.shop.form_login' shared service.

include_once \dirname(__DIR__, 4).'/vendor/symfony/security/Http/Authentication/AuthenticationSuccessHandlerInterface.php';
include_once \dirname(__DIR__, 4).'/vendor/symfony/security/Http/Authentication/CustomAuthenticationSuccessHandler.php';

return $this->privates['security.authentication.success_handler.shop.form_login'] = new \Symfony\Component\Security\Http\Authentication\CustomAuthenticationSuccessHandler(($this->privates['sylius.authentication.success_handler'] ?? $this->load('getSylius_Authentication_SuccessHandlerService.php')), ['login_path' => 'sylius_shop_login', 'default_target_path' => 'sylius_shop_homepage', 'use_referer' => true, 'always_use_default_target_path' => false, 'target_path_parameter' => '_target_path'], 'shop');
